React/README.md

```diff
@@ -1,102 +1,100 @@
-# React Challenge
+# React challenges
 
+Not everything is important, but if you're in doubt, you can check our [evaluation criteria](../EvaluationCriteria.md).
 
-## 1 - Mern-seed  login research:
-Clone the mern-seed project.  
-Create a branch from master following our github conventions in case you want to implement the feature or just add comments/suggestions of the code that you look.
-If we decided to implement a login form into the project, what will be the best approach to take? You can create a comparison table of the possible options and make a conclussion with what will be for you the best decision to take (assume that the client has an amazing API)
+## 1 - MERN-Seed login research:
 
-Bonus track: implement the login based on your decision and made a PR with your changes
+Clone the [MERN-Seed][] project.
```

Comment:
I-'ll create a ticket since mern-seed is deprecated in favor to https://github.com/MakingSense/react-redux-starter-app

Reply:
You could have just asked me to change the link to the new project.